src/powershell-native/nativemsh/pwrshcommon/pwrshcommon.cpp

```diff
@@ -668,12 +668,14 @@ namespace NativeMsh
     // NOTE: The names must not include the .dll extension because it will be added programmatically.
     static PCSTR trustedAssemblies[] =
     {
+        "getfilesiginforedistwrapper",
```

Comment:
What is this?

Reply:
The new list is generated from the trace output of pwsh.exe (the min-size package, so WPF things are not included).
For this specific one, I don't know where it's pulled from.